### PR TITLE
chore(deps): eslint 7 + new rules

### PR DESCRIPTION
**Pull request by e110c0**
* update to eslint 7
* added new rules
* removed rules now present in airbnb config
* eslint-disables require a description now in the same line

this will be published as a major version due to the changed required versions

### CODE REVIEW

File: packages/eslint-config-react/rules/react-intl.js [hunk 1]
```diff
@@ -72,13 +72,6 @@ module.exports = {
     */
     'formatjs/no-offset': ['error'],
 
-    /*
```
Comment (e110c0) [inline packages/eslint-config-react/rules/react-intl.js:63]: why did you remove this?

Comment (bitcloud) [reply packages/eslint-config-react/rules/react-intl.js:63]: image

It is not supported by upstream anymore

Comment (e110c0) [reply packages/eslint-config-react/rules/react-intl.js:63]: oh, that was a short-lived rule then :)
